pkg/nsx/services/common/types.go

```diff
@@ -40,9 +40,6 @@ const (
 	TagScopeNSXServiceAccountCRUID  string = "nsx-op/nsx_service_account_uid"
 	TagScopeNSXProjectID            string = "nsx-op/nsx_project_id"
 	TagScopeProjectGroupShared      string = "nsx-op/is_nsx_project_shared"
-	TagScopeVPCPodNamespaceUID      string = "nsx-op/subnet_namespace_uid"     // Todo: pod subnet, need to change according to subnet controller
-	TagScopeVPCVMNamespaceUID       string = "nsx-op/subnet_vif_namespace_uid" // Todo: VM subnet, need to change according to subnet controller
-	TagScopeVPCPod                  string = "nsx-op/pod_uid"                  // Todo: pod subnetport, need to change according to pod controller
```

Comment:
It seems that you lost `TagScopePodUID string = "nsx-op/pod_uid"` in this patch set?